lib/src/index.js

```diff
@@ -23,6 +23,10 @@ function src(glob, opt) {
   opt.readFile = opt.read;
   opt.read = undefined;
 
+  // Leverage node-glob's statCache, conservatively
+  // TODO allow users to provide this option, at their own risk?
+  opt.statCache = Object.create(null);
```

Comment:
Mind opening it as a separate PR and I'll flag it as "help wanted" and ask for help writing more tests.